exec/testadjustcf.R

```diff
@@ -189,8 +190,8 @@ if (args$maxrecs > 0) {
   dm_filt <- dm
 }
 
-# Create outdir if necessary; ignore warnings if it already exists
-dir.create(file.path(outdir), showWarnings = FALSE)
+# # Create outdir if necessary; ignore warnings if it already exists
```

Comment:
Should these two commented lines be removed?

Reply:
Yes! Will do, thanks for catching that!